engine/src/main/java/com/agiletec/aps/system/services/i18n/I18nManagerWrapper.java

```diff
@@ -43,9 +59,15 @@ public I18nManagerWrapper(String langCode, II18nManager i18nManager) {
 	 */
 	public String getLabel(String key) throws EntException {
 		String label = null;
-		if (key != null) {
-			label = this._i18nManager.getLabel(key, this._lang);
+		if (null != key) {
```

Comment:
IMHO the use of functional+Optional in this section with all this checks will make the code easy to read simple and null pointer proof

Reply:
Converted